### ISSUE TITLE
Tối ưu siêu tham số trên bộ data ver 3

### ISSUE THREAD

**Issue by yenq89** [yenq89/SE2025-14.2#31]
### Ver 3 (~5k ảnh) thử với các bộ tham số: 

| Phiên bản | train_batch_size | num_train_epochs | learning_rate | lr_scheduler | 
|----------|------------------|------------------|---------------|--------------|
| Config A | 8                | 20               | 1e-4          | constant     | 
| Config B | 16               | 40               | 1e-4          | constant     | 

### Đề xuất thêm

| Phiên bản | train_batch_size | num_train_epochs | learning_rate | lr_scheduler |
|----------|------------------|------------------|---------------|--------------|
| Config C | 8                | 30               | 5e-5          | cosine       | 

Hoặc nếu có đề xuất thêm, ghi thêm vào comment trên issue này. 

Nhận xét loss, validation trực tiếp vào issue và tổng hợp ngắn gọn trên report tuần.

**Comment by yenq89** [comment]
Config A:

``` 
export MODEL_NAME="runwayml/stable-diffusion-v1-5"
export DATASET_NAME="/mnt/c/Users/ssm_d/SE2025-14.2/data_ver3/ghibli"

accelerate launch --num_processes=1 --mixed_precision="no" train_text_to_image_lora.py \
  --pretrained_model_name_or_path="$MODEL_NAME" \
  --train_data_dir="$DATASET_NAME" \
  --caption_column="text" \
  --resolution=512 \
  --random_flip \
  --train_batch_size=8 \
  --num_train_epochs=20 \
  --checkpointing_steps=1000 \
  --learning_rate=1e-04 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --seed=42 \
  --output_dir="/mnt/c/Users/ssm_d/SE2025-14.2/model/ghibli_checkpoint_data_v3" \
  --validation_prompt="Ghibli style a young girl with brown hair tied back, wearing a simple red dress" \
  --report_to="wandb"
```
